src/build.sh

```diff
@@ -89,12 +89,18 @@ function kernel_build()
   fi
 
   if [ -x "$(command -v nproc)" ]; then
-    parallel_cores=$(nproc --all)
```

Comment:
Since you are working on this part of the code, we can replace `-x "$(command ...` with `command_exists`.